CMake/freud-macros.cmake

```
@@ -33,3 +33,17 @@ function(make_python_extension_module _target)
       "$<$<PLATFORM_ID:Darwin>:LINKER:-undefined,dynamic_lookup>")
   endif()
 endfunction()
+
+# set the rpath for installing python extension modules
+function(target_set_install_rpath _target)
+  if(APPLE)
+    set_target_properties(${_target} PROPERTIES INSTALL_RPATH "@loader_path")
+  else()
+    set_target_properties(${_target} PROPERTIES INSTALL_RPATH "\$ORIGIN")
+  endif()
+
+  if(_using_conda OR DEFINED ENV{CIBUILDWHEEL})
```

```suggestion
  if(_using_conda)
```
I modified the cibuildwheel action to set `INSTALL_RPATH_USE_LINK_PATH`.